dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use an official Node.js runtime as a parent image
+FROM node:14-alpine
```

Comment:
Node 14 is now EOL. The most recent LTS version is node:18.

Reply:
I'm also developing and testing on 18 👍

Reply:
ok min version is 16, I just added nvmrc file and `engines` prop in the `package.json`